packages/react-ui-kit/src/Form/Select.tsx

```diff
@@ -31,16 +31,17 @@ import type {Theme} from '../Layout';
 import InputLabel from './InputLabel';
 
 export type Option = {
-  value: string | number;
+  value: string;
   label: string;
   description?: string;
   isDisabled?: boolean;
 };
 
-interface SelectProps<IsMulti extends boolean> extends StateManagerProps<Option, IsMulti> {
+interface SelectProps<IsMulti extends boolean> extends Omit<StateManagerProps<Option, IsMulti>, 'onChange'> {
   id: string;
   dataUieName: string;
   options: Option[];
+  onChange: (value: Option) => void;
```

Comment:
Isn't the value `Option[]` when select is `IsMulti`?

Reply:
that is incorrect from my testing.. for multi select it passes all

Reply:
that's what I assumed as well when talking to @przemvs